frontend/src/v5/ui/components/dashboard/dashboardList/dasboardList.styles.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled, { css } from 'styled-components';
+
+export const ListContainer = styled.ul`
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	margin: 0;
+	padding: 0;
+`;
+
+export const DashboardListHeader = styled.div`
+	display: flex;
+	padding: 13px 20px 13px 30px;
+`;
+
+export const FixedOrGrowContainer = styled.div`
+  flex: 1;
+	
+  ${({ width }) => width && css`
+    min-width: ${width}px;
+    flex: 0;
+  `}
+`;
```

Comment:
I think this should go in the controls folder. In that case I think typing it would be helpful because it would be shared across different components.